### DISCUSSION TITLE
Progetto esame Open Source - Michal Horowski

### DISCUSSION THREAD

**Discussion by mibbbbbnPG** [deneb-alpha/Lab_open_source-2023_2024#31] [Projects]
MAIN
    Test & Quality Assurance(QA) di LibreOffice
    Scopo di questo settore
    Come viene svolto
    La mia esperienza
EXTRA
    Open source Hardware: in particolare il progetto Arduino

Proposta temi esame Michal Horowski.

Buonasera professoressa, vorrei inviarle la mia proposta d'esame. Come può vedere la parte extra è un po' particolare, ciononostante ho voluto suggerirla in quanto l'argomento mi affascina e ho visto molte analogie con le idee che stanno alla base del Software open source.

La ringrazio in anticipo, cordiali saluti Michal Horowski.

**Comment by deneb-alpha** [comment]
l'open hardware ha moltissimo in comune con il free software. Sono curiosa di vedere cosa presenterai per quella sezione extra!
